docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,556 @@
+
+**NOTE:** This requires Windows and .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+Targeting a PCL profile is a bit trickier.  For starters, [reference this list of PCL profiles](http://embed.plnkr.co/03ck2dCtnJogBKHJ9EjY/preview) to ensure you have the correct target.  Hover over the Name of each entry for the framework identifier, which you will need.
+
+You will then need to do two things:
+
+1. List the dependencies for each target *inside* of that target.  The "global" `dependencies` section cannot be used.
+2. List the framework assemblies you are using in your code *inside* the PCL target profile section.  This will require, at a minimum, `mscorlib`, `System`, and `System.Core`.
+
+Here is an example targeting PCL Profile 328 and .NET Core, using only `System.Runtime` as a dependency for .NET Core.
+
+```
+{
+    "frameworks":{
+        "dotnet55":{
+           "dependencies":{
+                "System.Runtime":"4.0.0-rc1-*"
+            }
+        },
+        ".NETPortable,Version=v4.0,Profile=Profile328":{
+            "frameworkAssemblies":{
+                "mscorlib":"",
+                "System":"",
+                "System.Core":""
+            }
+        }
+    }
+}
+```
+
+This will allow you to target .NET Platform 5.5, .NET Framework 4.0, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8.1, and Silverlight 5.0.
+
+You can now build.
+
+Using DNX:
+
+```
+$ dnu restore
+$ dnu build
+```
+
+Using .NET CLI:
+
+```
+$ dotnet restore
+$ dotnet compile
+```
+
+You will now notice three new entries in your `/bin/Debug` folder:
+
+```
+/bin
+   /Debug
+      /dotnet55
+      /portable-net40+sl50+netcore45+wpa81+wp8
+```
+
+Each folder contains distinct `.dll`s corresponding to their named target.
+
+**NOTE:** This assumes your code will compile across *both* .NET Core and .NET Framework.  Read the section on cross-compiling with `#if`s on how to compile the same file differently for each target if you are using features which are unavailable in some of your targets.
+
+## How to cross-compile for .NET Core and .NET Framework
+
+**NOTE:** This requires Windows and .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+If you need to support .NET Framework and .NET Core, you may find yourself needing to use different APIs to accomplish the same task.  You may also wish to support different language constructs (such as `async` and `await`, which only became available with .NET Framework 4.5) and support modern APIs within the same file.  A great way to do this is to cross-compile with `#if` guards.
+
+For example, in .NET Core you can use `HttpClient` in the `System.Net.Http` API to perform network operations over Http.  However, the .NET 4.0 Framework does not include `HttpClient`!  Instead, you may need to use `WebClient` from the `System.Net` assembly.
```

Comment:
Suggested rewriting: "For example, in .NET Core, you can use the `HttpClient` class from the `System.Net.Http` namespace to perform network operations over HTTP. However, the .NET Framework 4.0 doesn't contain an `HttpClient` class (which is only available starting with .NET Framework 4.5), so you may need to use the `WebClient` class from the `System.Net` namespace instead."